test/UnitTests/MSTest.Analyzers.UnitTests/UseProperAssertMethodsAnalyzerTests.cs

```diff
@@ -354,10 +582,28 @@ public void MyTestMethod()
             }
             """;
 
-        await VerifyCS.VerifyAnalyzerAsync(
+        // error CS1503: Argument 1: cannot convert from 'object' to 'bool'
```

Comment:
Shall we instead not raise? We could detect if that can be converted to the other type. I guess